WORKSPACE

```
@@ -43,30 +43,21 @@ load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kotlin_repositories", "kt_reg
 kotlin_repositories()
 kt_register_toolchains()
 
-# rules_proto defines abstract rules for building Protocol Buffers.
-http_archive(
-    name = "rules_proto",
-    sha256 = "2490dca4f249b8a9a3ab07bd1ba6eca085aaf8e45a734af92aad0c42d9dc7aaf",
-    strip_prefix = "rules_proto-218ffa7dfa5408492dc86c01ee637614f8695c45",
-    urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/218ffa7dfa5408492dc86c01ee637614f8695c45.tar.gz",
-        "https://github.com/bazelbuild/rules_proto/archive/218ffa7dfa5408492dc86c01ee637614f8695c45.tar.gz",
-    ],
+'''
+The proto_compiler and proto_java_toolchain bindings load the protos rules needed for the model
+module while helping us avoid the unnecessary compilation of protoc. Referecences:
+https://docs.bazel.build/versions/master/be/workspace.html#bind,
+https://github.com/google/startup-os/blob/master/WORKSPACE#L215-L220,
+https://github.com/bazelbuild/bazel/issues/7095
```

```suggestion
- https://docs.bazel.build/versions/master/be/workspace.html#bind
- https://github.com/google/startup-os/blob/master/WORKSPACE#L215-L220
- https://github.com/bazelbuild/bazel/issues/7095
```

Markdown is a nice way to enumerate items in a readable way.

---

Done!